docs/advanced/workers.md

```diff
@@ -11,18 +11,18 @@ This feature is <b>experimental</b> and should only be used when you are reachin
 
 You must first have a working Redis instance somewhere which can talk between processes. For example, in Docker you can run:
 
-`docker run --name github-bridge-redis -p 6379:6379 -d redis`.
+`docker run --name redis-host -p 6379:6379 -d redis`.
 
 The processes should all share the same config, which should contain the correct config to enable Redis:
 
 ```yaml
 queue:
-  monolithic: false
-  port: 6379
-  host: github-bridge-redis
+  redisUri: "redis://redis-host:6379"
+cache:
```

Comment:
Is this needed here? As it's already mentioned below.

Reply:
I'm going to keep it out of caution, as it could look like `queue` replaces `cache` having the same properties. Specifying it twice does no harm either.